src/lm.jl

```diff
@@ -119,11 +122,18 @@ const FIT_LM_DOC = """
     is less-than-full rank. If `true` (the default), only the first of each set of
     linearly-dependent columns is used. The coefficient for redundant linearly dependent columns is
     `0.0` and all associated statistics are set to `NaN`.
+
+    `contrasts` may optionally be supplied in the form of a `Dict` mapping term names
+    (as `Symbol`s) to term or contrast types. If a contrast is not provided
```

Comment:
migth be good to give a few examples; IIRC you can't pass a contrast _type_ (like `HelmertCoding`) but need an instantiated contrasts struct (like `HelmertCoding()`).
```suggestion
    (as `Symbol`s) to term types (e.g. `ContinuousTerm`) or contrasts 
    (e.g., `HelmertCoding()`, `SeqDiffCoding(; levels=["a", "b", "c"])`, 
    etc.). If contrasts are not provided
```